### PR TITLE
Feature/ts expect error

### PR DESCRIPTION
**Pull request by boazpoolman**

### CODE REVIEW

**Comment by boazpoolman** [comment]
@Alexnortung Could you review this PR also? :)

File: playground/types/generated/contentTypes.d.ts
Comment (Alexnortung) [inline playground/types/generated/contentTypes.d.ts:1]: Not sure if this is intentional deletion

File: packages/core/server/admin-api/migrations/plugin-options-rename.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ const migratePluginOptionsRename = (strapi: IStrapi) => {
     delete updatedContentType.pluginOptions['url-alias'];
 
     // Set the new options
-    // @ts-ignore
+    // @ts-expect-error
```
Comment (Alexnortung) [inline packages/core/server/admin-api/migrations/plugin-options-rename.ts:19]:
```suggestion
    // @ts-expect-error - TODO: extend pluginOptions type
```

**Review by Alexnortung — CHANGES_REQUESTED** [review]
only thing I want to comment on is adding a description for why we expect an error, for example: `// @ts-expect-error - Invalid generated types`. Then it will be easier when we need to removed the comments if we did actually resolve the original error.

**Pull request closed without merging**